Tasks/AzureRmWebAppDeployment/operations/ContainerBasedDeploymentUtility.ts

```diff
@@ -22,7 +22,7 @@ export class ContainerBasedDeploymentUtility {
     }
 
     public async deployWebAppImage(taskParameters: TaskParameters): Promise<void> {
-        var imageName = this._getImageName();
+        let imageName: string = this._getDockerHubImageName();
```

Comment:
Since we are jumping task major version here. There is an opportunity for us to bring docker tasks and azure web app task in sync with how they take image name as input. in Docker tasks there is only one input image name which includes repository name and tag while azure weapp task takes them separately.

Reply:
I thought the reason these are separate is because at some point we want to enable drop down for each of these inputs, which will not be possible if we take them as a single input.